composer.json

```diff
@@ -21,7 +21,7 @@
         "symfony/yaml": "~2.3|~3|~4",
         "symfony/phpunit-bridge": "~2.7|~3|~4",
         "phpunit/phpunit": "^5.7.1|~6|~7",
-        "cucumber/cucumber": "dev-gherkin-16.0.0"
+        "cucumber/cucumber": "dev-gherkin-15.0.1"
```

Comment:
This was to make sure the script did something, but I could actually commit it to master and wait a day, then we'd see a new PR the next morning if it's all working ok?